libs/domains/observability/feature/src/lib/service-overview/card-http-errors/card-http-errors.tsx

```diff
@@ -0,0 +1,69 @@
+import { useState } from 'react'
+import { useMetrics } from '../../hooks/use-metrics/use-metrics'
+import { CardMetric } from '../card-metric/card-metric'
+import { InstanceHTTPErrorsChart } from '../instance-http-errors-chart/instance-http-errors-chart'
+import { ModalChart } from '../modal-chart/modal-chart'
+import { useServiceOverviewContext } from '../util-filter/service-overview-context'
+
+export function CardHTTPErrors({ serviceId, clusterId }: { serviceId: string; clusterId: string }) {
+  const { timeRange } = useServiceOverviewContext()
+  const [isModalOpen, setIsModalOpen] = useState(false)
+
+  const { data: metrics, isLoading: isLoadingMetrics } = useMetrics({
+    clusterId,
+    query: `100 *
+      sum(
+        rate(nginx_ingress_controller_requests{status!~"2.."}[${timeRange}])
+          * on(ingress) group_left(label_qovery_com_associated_service_id)
+            max by(ingress, label_qovery_com_associated_service_id)(
+              kube_ingress_labels{label_qovery_com_associated_service_id =~ "${serviceId}"}
+            )
+      )
+      /
+      clamp_min(
+        sum(
+          rate(nginx_ingress_controller_requests[${timeRange}])
+            * on(ingress) group_left(label_qovery_com_associated_service_id)
+              max by(ingress, label_qovery_com_associated_service_id)(
+                kube_ingress_labels{label_qovery_com_associated_service_id =~ "${serviceId}"}
+              )
+        ),
+        1
+      ) or vector(0)`,
```

Comment:
Nitpick: It would be nice to extract the queries to a function that would be declared at the top of the file. 
What do you think?